### ISSUE TITLE
Cannot read property 'urlToPathAndParams' of undefined Evaluating App.js Loading App.js

### ISSUE THREAD

**Issue by TanishqSinghAnand** [expo/snack#35]
While using the navigation in Snack it works at the first instance and after few days when we open the project it shows the error "Cannot read property 'urlToPathAndParams' of undefined Evaluating App.js Loading App.js" in line App.js(0,1)
And if we use it on our local machines it works.

And this is not in only 1 but in all projects.

**Comment by IjzerenHein** [comment]
Hi. Please provide a working Snack that exhibits the problem.

**Comment by TanishqSinghAnand** [comment]
> Hi. Please provide a working Snack that exhibits the problem.

https://snack.expo.io/@tanishq0504/283cc9

**Comment by IjzerenHein** [comment]
Hi! I've fixed the problem. The `@react-naviation/native@3.8.3`package was not generated correctly because it did not have `@react-navigation/core` listed in the `dependencies` in `package.json`. Instead it was listed in `devDependencies`. I've updated the Snackager bundler to deal with these kinds of situations and regenerated the bundle.
It now works, but you might need to clear the cache in your browser to fetch the latest bundle and get it to work:

![image](https://user-images.githubusercontent.com/6184593/103655487-9ed64380-4f67-11eb-8817-136fe7e8eb2c.png)

And when using the Expo client on your iPhone or Android device, you might need to uninstall and re-install it in order to clear its cache, otherwise it might use the old faulty bundle.
